frontend/src/features/map_planning/routes/Map.tsx

```diff
@@ -42,8 +49,116 @@ export const Map = () => {
+        <Toolbar
+          minWidth={160}
+          contentTop={
+            <div>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
+              <IconButton className="m-2 h-8 w-8 border border-neutral-500 p-1">
+                <PlantIcon></PlantIcon>
+              </IconButton>
```

Comment:
That's a lot of code duplication, I think it would be better to use a `map`

Reply:
@buenaflor Thx for the review! Are the other parts okay, could you continue to work on #319 if this gets merged? (Please approve/request changes to clarify.)

Reply:
You are right but those are just placeholder buttons to review the layout. Later on they will all have their own icon and click handlers and then we should probably extract them to a new component.